components/indexComponents/scrollUp.tsx

```diff
@@ -0,0 +1,35 @@
+import { motion } from "framer-motion";
+import { useCallback, useEffect, useState } from "react";
+import { VscChevronUp } from "react-icons/vsc";
+import styles from "@/styles/pages.module/companies.module.scss";
+
+export default function ScrollUp() {
+  const [scrollY, setScrollY] = useState(0);
+  const onScroll = useCallback((event: any) => {
```

Comment:
Używanie `any` neguje wszystko to, w czym TypeScript jest tak dobry. Zawsze istnieje odpowiedni typ, który można przypisać - w tym wypadku wbudowany typ `Event`.

```suggestion
  const onScroll = useCallback((event: Event) => {
```